Gemfile

```diff
@@ -23,6 +22,9 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
+gem 'bootstrap-sass', '~> 3.2.0'
```

Comment:
Yea its just basically a different pre-processor for CSS and you can use regular CSS with it if you want.